### DISCUSSION TITLE
How to disable hover effect?

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [anyrun-org/anyrun#71] [Q&A]
![grim-30-07-2023-14-29-17](https://github.com/Kirottu/anyrun/assets/20235646/036878f1-3f00-4c29-8832-4a442f0fb459)

How can I disable the hover effect that adds a grey-ish border around the presently hovered over group of plugin matches? In this case, the Symbols plugin.

**Comment by Kirottu** [comment]
This can be done via CSS styling, iirc my Anyrun style in my dotfiles repo disables this effect.

**Comment by lokesh-krishna** [comment]
I did check your dotfiles for ideas and your styling too has the hover effect, except that it applies to the entire area rather than as a border. 

![image](https://github.com/Kirottu/anyrun/assets/20235646/7e1e422d-eaff-42f9-b90c-68116386c928)

Here's what it looks like with your `style.css`.

**Comment by lokesh-krishna** [comment]
Has anyone else been able to figure out where that hover effect is coming from?

**Reply by zeishr** [reply]
You can use `#match:selected` and `#match:hover` to change selected/hover effects

**Reply by TheFrankyDoll** [reply]
Only way I found was adding 

```
* {
    all: unset;
  }
```
to the the top of the file. Looks like something in GTK theme sets this border.
